### ISSUE TITLE
Meeting Details JS Error

### ISSUE THREAD

**Issue by klgrimley** [bmlt-enabled/crouton#442]
**Describe the bug**
One subsite in a multisite network gets a JS error when trying to access a meeting details page

**To Reproduce**
1. Go to [https://naindiana.org/michiana-area/meetings/?meeting-id=10&language=en-US&time_format=h%3Amm%20a](https://naindiana.org/michiana-area/meetings/?meeting-id=10&language=en-US&time_format=h%3Amm%20a)

**Expected behavior**
Page loads as in the rest of the network

**Error Message**
```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'url')
    at Crouton.t.enrichMeetings (crouton-core.min.js?ver=1699817995:1:344902)
    at crouton-core.min.js?ver=1699817995:1:349884
```

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Brave & Chrome

**Smartphone (please complete the following information):**
 - Device: iPhone
 - OS: iOS 17
 - Chrome

**Comment by otrok7** [comment]
Need to clear extra_meetings before generating the meeting-details page.  In this case, the extra meetings are in a different Service Body, but we never retrieved the data for these service bodies, only for the target of the meeting-details.  Will be fixed in next release.

**Comment by otrok7** [comment]
Fixed in 3.17.4
